google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java

```diff
@@ -226,7 +230,14 @@ static Finding setFindingState(FindingName findingName) {
   // [START securitycenter_list_all_findings]
   static ImmutableList<ListFindingsResult> listAllFindings(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
+      // SourceName must be in one of the following formats:
```

Comment:
Should this really say "SourceName"? The list that follows seems to show only the format of the org, folder, or project specification.

Reply:
Done. Thanks